frontend/src/scenes/experiments/charts/funnel/SampledSessionsModal.tsx

```diff
@@ -41,7 +41,7 @@ const getLinkTextAndUrl = (session: SessionData): { text: string; url: string }
     } else if (session.person_id) {
```

Comment:
just above here if there's a `session.session_id` we go to the activity tab instead based on `session_id`. As noted in the PR description I find this a bit confusing, especially if you're used to Product Analytics funnels which just link to the person.